### PR TITLE
trapezoidal fuzzy set 

### PR DESCRIPTION
**Pull request by Shreya123714**
### Describe your change:
I created a trapezoidal fuzzy set, which will be further used in a fuzzy inference system. 
I plan to add the algorithm for the fuzzy inference system in another PR.

* [X] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Add or change doctests? -- Note: Please avoid changing both code and tests in a single pull request.
* [ ] Documentation change?

### Checklist:
* [X] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [X] This pull request is all my own work -- I have not plagiarized.
* [X] I know that pull requests will not be merged if they fail the automated tests.
* [X] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [X] All new Python files are placed inside an existing directory.
* [X] All filenames are in all lowercase characters with no spaces or dashes.
* [X] All functions and variable names follow Python naming conventions.
* [X] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [X] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [X] All new algorithms include at least one URL that points to Wikipedia or another similar explanation.
* [X] If this pull request resolves one or more open issues then the description above includes the issue number(s) with a [closing keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue): "Fixes #ISSUE-NUMBER".

### CODE REVIEW

File: fuzzy_logic/trapazoidal_fuzzyset.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        """
+        Creates a new TrapezoidalFuzzySet instance representing the complement.
+        
+        >>> TrapezoidalFuzzySet("Medium", 0, 1, 2, 3).complement().membership(1)
+        0.0
+        """
+        return TrapezoidalFuzzySet(f"¬{self.name}", self.left_base, self.left_peak, self.right_peak, self.right_base,
+                                   is_complement=not self.is_complement)
+
+    def plot(self) -> None:
```
[the remark this answers is not in the document]
Comment (Shreya123714) [reply fuzzy_logic/trapazoidal_fuzzyset.py:85]: ok

**Review by Shreya123714 — COMMENTED** [review]
have added a doctest  for plot()

File: fuzzy_logic/trapazoidal_fuzzyset.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (Shreya123714) [reply fuzzy_logic/trapazoidal_fuzzyset.py:85]: > As there is no test file in this pull request nor any test function or class in the file `fuzzy_logic/trapazoidal_fuzzyset.py`, please provide doctest for the function `plot`

I have added it please check

**Comment by cclauss** [comment]
Closing require_tests PRs to prepare for Hacktoberfest

**Pull request closed without merging**